src/EditModel/EditDataEntryForm.component.js

```diff
@@ -162,6 +162,7 @@ class EditDataEntryForm extends Component {
 
             const formHtml = dataSet.dataEntryForm ? this.processFormData(dataSet.dataEntryForm) : '';
 
+            // TODO
```

Comment:
There's some code below this line in the `setState` callback that creates a ckeditor editor using window.CKEDITOR. This will have to be replaced with a call to the `useEditor` hook from `ckeditor4-react` as the `import 'ckeditor'` import that adds window.CKEDITOR does not seem to load files needed by ckeditor correctly.

Reply:
We talked about this. This is the reason why the PR is still in draft state, right? So it's just a reminder for you and won't stay in the codebase once the PR is ready / merged

Reply:
Exactly there's a big chunk of this PR left to do and once the PR is merged there shouldn't be any TODO comments left.